### PR TITLE
Pre pull local-volume-provisioner image, improve quay outage sensitivity.

### PR DESCRIPTION
**Pull request by oshoval**
Makes the providers less sensitive to quay outage,
since we pre pull the images upon provisioning,
this image was missing.

Affected K8s-1.17, K8s-1.18 providers

Signed-off-by: Or Shoval <oshoval@redhat.com>

### CODE REVIEW

**Comment by oshoval** [comment]
@rmohr 
Can you review this please?
Its meant to allow us cluster-up even when quay has outage, as groomed once.

Do we want to do it also for 1.18 i guess ?

i provisioned new 1.17
9f31e4580c854a7fcc6f4a044217895217aa29ddeb5b3346205a9008c06ada0d
will update the hash once its lgtm by you, and will update 1.18 as well if needed, thanks

File: cluster-provision/k8s/1.17/provision.sh [hunk 1]
```diff
@@ -322,6 +322,9 @@ docker pull quay.io/k8scsi/csi-node-driver-registrar:v1.0.2
 cp -rf /tmp/cnao/ /opt/
 for i in $(grep -A 2 "IMAGE" /opt/cnao/operator.yaml |grep value | awk '{print $2}'); do docker pull $i; done
 
+# Pre pull local-volume-provisioner
+grep -A 2 "IMAGE" /tmp/local-volume.yaml | grep value | awk '{print $2}' | xargs docker pull
```
Comment (phoracek) [inline cluster-provision/k8s/1.17/provision.sh:326]: Is this needed in 1.18 too?

Comment (oshoval) [reply cluster-provision/k8s/1.17/provision.sh:326]: yep, mentioned it in the PR, 
unless 1.18 doesnt have local-volume-provisioner,
i will verify,
question out of the scope of the PR, is it fine that we dont have common.sh for k8s ?
and every change we need to duplicate ? 
its prune to errors.

Comment (oshoval) [reply cluster-provision/k8s/1.17/provision.sh:326]: updated 1.18, its needed as well
if it looks good i will updated hashes, thanks

**Comment by qinqon** [comment]
/lgtm

@oshoval can you update shas ?

**Comment by oshoval** [comment]
> /lgtm
> 
> @oshoval can you update shas ?
sure on it, waited for lgtm

**Comment by oshoval** [comment]
Updated hashes, will be glad if you can take another look please.
Thanks

**Pull request merged**